### PR TITLE
#158792176 Refactor Error Message

### PR DESCRIPTION
**Pull request by GbengaOshinaga**
#### Screenshots (if appropriate)
screen shot 2018-07-04 at 10 00 06 pm

### CODE REVIEW

File: api/models/model_operations.py [hunk 1]
```diff
@@ -58,9 +58,13 @@ def delete(self):
             db.session.add(self)
             db.session.commit()
         else:
+            relationship_names = []
+            for relationship in relationships:
+                relationship_names.append(relationship.first().__class__.__name__ + '(s)')
```
Comment (ibutiti) [inline api/models/model_operations.py]: Please use f strings

File: api/models/asset_category.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ class AssetCategory(AuditableBaseModel):
                                  cascade='save-update, delete', lazy='dynamic')
 
     def get_child_relationships(self):
-        return self.assets, self.attributes
+        return (self.assets,)
```
Comment (ibutiti) [inline api/models/asset_category.py]: We need to check for non-deleted child attributes too. Is there a reason you've removed this?

Comment (SEUNAGBEYE) [reply api/models/asset_category.py]: @ibutiti I believe an asset category should be deletable even if it has non-deleted child attributes when an asset category is deleted all its non-deleted child attributes should be set to `True`(Just like CASCADE on delete). I believe a more efficient way is to make the relationship between `asset category` and `attributes` `many to many`, this way attributes can be sharable among asset categories

File: api/models/model_operations.py (hunk 1 above)
Comment (GbengaOshinaga) [reply api/models/model_operations.py]: On it

**Comment by GbengaOshinaga** [comment]
@SEUNAGBEYE Done

File: api/models/asset_category.py (hunk 2 above)
Comment (ibutiti) [reply api/models/asset_category.py]: Cool @GbengaOshinaga @purpose50 @SEUNAGBEYE

Comment (brayoh) [reply api/models/asset_category.py]: Remove the comma at the end

Comment (GbengaOshinaga) [reply api/models/asset_category.py]: @brayoh According to the implementation, this function must return a tuple

Comment (brayoh) [reply api/models/asset_category.py]: Cool

**Pull request merged**